tools/serverpod_cli/lib/src/analyzer/entities/converter/converter.dart

```diff
@@ -5,6 +5,10 @@ import 'package:yaml/yaml.dart';
 // ignore: implementation_imports
 import 'package:yaml/src/equality.dart';
 
+List<String> convertIndexList(String stringifiedFields) {
```

Comment:
You are right! I forgot to add this in the parser, I wanted it to be here since the logic is the same in the parser and the validator. I will add this in the parser as well.